.vscode/extensions.json

```diff
@@ -0,0 +1,26 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
```

Comment:
Very strange... comments are not allowed in json. Apparently .vscode uses it's own json rules, ew... 😝